### DISCUSSION TITLE
Syntax error in tag 'highlight' when building via Github pages (remote theme)

### DISCUSSION THREAD

**Discussion by rdewdrops** [mmistakes/so-simple-theme#448] [Q&A]
Hi,

I am running into the following error when building the site via Github pages:
`Liquid Exception: Syntax Error in tag 'highlight' while parsing the following markup: Valid syntax: highlight <lang> [linenos] in CHANGELOG.md`

I have the following changes made to my `_config.yml` and `Gemfile` respectively, as mentioned in the docs:

**Config.yml**
`remote_theme: "mmistakes/so-simple-theme"`

**Gemfile**
```
source "https://rubygems.org"
gem "github-pages", group: :jekyll_plugins
gem "tzinfo-data"
gem "wdm", "~> 0.1.0" if Gem.win_platform?
```

**Comment by mmistakes** [comment]
The error above tells you what file Jekyll is having problems (a reference to `{% highlight %}` tag in `CHANGELOG.md`.

Looks like you didn't [remove some unnecessary files](https://github.com/mmistakes/so-simple-theme#remove-the-unnecessary) used only in the maintenance of the theme. Purge those from your repo and you should be good to go.
